app/assets/stylesheets/_variables.scss

```diff
@@ -16,20 +16,54 @@ $small-spacing: $base-spacing / 2;
 $base-z-index: 0;
 
 // Colors
-$blue: #477dca;
-$dark-gray: #333;
-$medium-gray: #999;
-$light-gray: #ddd;
-
-// Font Colors
-$base-background-color: #fff;
-$base-font-color: $dark-gray;
-$action-color: $blue;
-
-// Border
-$base-border-color: $light-gray;
+$dark-color: #333;
```

Comment:
Because a dark color can be a dark bluish tone or such. The idea is to keep it completely generic.